apps/src/aichat/redux/slice.ts

```diff
@@ -82,7 +82,6 @@ const aichatSlice = createSlice({
       }
 
       if (lastResetIndex >= 0) {
-        state.chatEventsPast = action.payload.slice(0, lastResetIndex + 1);
```

Comment:
I wonder if it would be useful for the user to see the clear chat notification to remind them that they took that user action, especially if they leave and then go back to the level.

Reply:
Yeah I was torn on this -- the clear chat event seems like it would be somewhat clear to users, but changing the model ID, temperature, etc also resets the conversation, which feels like it would be less clear to see at the top of the conversation? [Checking in Slack here.](https://codedotorg.slack.com/archives/C06FELVTV0Q/p1742833107390359?thread_ts=1742482983.291809&cid=C06FELVTV0Q)

Reply:
Oh interesting - I didn't realize that we're clearing back to an event that clears the chat context.

I wonder if it would make sense to add a reminder within the model customization update:
https://github.com/code-dot-org/code-dot-org/blob/dfe8d1c193d6af68acb33da457389e9e49475cb1/apps/i18n/aichat/en_us.json#L82-L83

Maybe something like, `'Reminder that your chat history has been cleared.'`?
cc: @dju90

Reply:
Updated to include the clear event, per Slack discussion.

I think there are a handful of changes/improvements that could be made now that we're showing history across page loads to users (I just jotted down a few, will share in Slack later). Maybe we could include this suggestion in that discussion?